### PR TITLE
docs: prepare v9 migration guide

### PR DESCRIPTION
**Pull request by matthewmayer**
alternative for #2788

concatenates the migration guide snippets. I tried to organize in a reasonably sensible order but didn't edit the content other than the headings.

EDIT: The merged page has been edited.

### CODE REVIEW

**Comment by matthewmayer** [comment]
Can you trigger the netlify deploy so we have an easy way to preview this?

**Comment by Shinigami92** [comment]
> Can you trigger the netlify deploy so we have an easy way to preview this?

You mean this? https://deploy-preview-2805.fakerjs.dev/

**Comment by matthewmayer** [comment]
> > Can you trigger the netlify deploy so we have an easy way to preview this?
> 
> You mean this? https://deploy-preview-2805.fakerjs.dev/

Thanks. https://deploy-preview-2805.fakerjs.dev/guide/upgrading.html

**Comment by ST-DDT** [comment]
IMO the `Remove deprecated locale aliases` is at the wrong location and should be maybe in front of https://deploy-preview-2805.fakerjs.dev/guide/upgrading.html#locale-aliases

File: docs/guide/upgrading.md [hunk 1]
```diff
@@ -10,4 +10,644 @@ This is the migration guide for upgrading from v8 to v9.
...
+| faker.image.placeholder.imageUrl          | faker.image.urlPlaceholder                             |
+| faker.image.placeholder.randomUrl         | faker.image.urlPlaceholder                             |
+| faker.image.unsplash.image                | faker.image.url                                        |
+| faker.image.unsplash.imageUrl             | faker.image.url                                        |
+| faker.image.unsplash.food                 | faker.image.urlLoremFlickr({ category: 'food' })       |
+| faker.image.unsplash.people               | faker.image.urlLoremFlickr({ category: 'people' })     |
+| faker.image.unsplash.nature               | faker.image.urlLoremFlickr({ category: 'nature' })     |
+| faker.image.unsplash.technology           | faker.image.urlLoremFlickr({ category: 'technology' }) |
+| faker.image.unsplash.objects              | faker.image.urlLoremFlickr({ category: 'objects' })    |
+| faker.image.unsplash.buildings            | faker.image.urlLoremFlickr({ category: 'buildings' })  |
```
Comment (xDivisionByZerox) [inline docs/guide/upgrading.md]: This is the only table which does not use backticks to highlight source code.

```suggestion
| old                                       | replacement                                            |
| ----------------------------------------- | ------------------------------------------------------ |
| `faker.image.lorempicsum.image`             | `faker.image.urlPicsumPhotos`                            |
| `faker.image.lorempicsum.imageGrayscale`    | `faker.image.urlPicsumPhotos({ grayscale: true })`       |
| `faker.image.lorempicsum.imageBlurred`      | `faker.image.urlPicsumPhotos({ blur: 4 })`               |
| `faker.image.lorempicsum.imageRandomSeeded` | `faker.image.urlPicsumPhotos`                            |
| `faker.image.lorempicsum.imageUrl`          | `faker.image.urlPicsumPhotos`                            |
| `faker.image.placeholder.imageUrl`          | `faker.image.urlPlaceholder`                             |
| `faker.image.placeholder.randomUrl`         | `faker.image.urlPlaceholder`                             |
| `faker.image.unsplash.image`                | `faker.image.url`                                        |
| `faker.image.unsplash.imageUrl`             | `faker.image.url`                                        |
| `faker.image.unsplash.food`                 | `faker.image.urlLoremFlickr({ category: 'food' })`       |
| `faker.image.unsplash.people`               | `faker.image.urlLoremFlickr({ category: 'people' })`     |
| `faker.image.unsplash.nature`               | `faker.image.urlLoremFlickr({ category: 'nature' })`     |
| `faker.image.unsplash.technology`           | `faker.image.urlLoremFlickr({ category: 'technology' })` |
| `faker.image.unsplash.objects`              | `faker.image.urlLoremFlickr({ category: 'objects' })`    |
| `faker.image.unsplash.buildings`            | `faker.image.urlLoremFlickr({ category: 'buildings' })`  |
```

**Comment by xDivisionByZerox** [comment]
@matthewmayer We applied some changes together in todays meeting. Please review them since we want to release an alpha version very soon.

**Review by xDivisionByZerox — APPROVED** [review]

**Review by ST-DDT — APPROVED** [review]

**Review by Shinigami92 — APPROVED** [review]
I would have suggestions, but nothing blocking
So we could do these changes in later iterations before doing an official release

**Comment by matthewmayer** [comment]
Same, I have minor feedback but looks great for now.

**Pull request merged**